id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -47,7 +130,8 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
             .namedValues(
               identifiers.column.CanonicalID -> identifier.CanonicalID,
               identifiers.column.ontologyType -> identifier.ontologyType,
-              identifiers.column.MiroID -> identifier.MiroID
+              identifiers.column.MiroID -> identifier.MiroID,
+              identifiers.column.CalmAltRefNo -> identifier.CalmAltRefNo
```

Comment:
What kind of extra tests would it need? I can see a bunch of Calm related tests below...

Reply:
It doesn’t, I realised they were already written…